modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/shibari_worn.dm

```diff
@@ -0,0 +1,268 @@
+	if(iscarbon(user))
+		var/mob/living/carbon/human/hooman = user
+		if(src == hooman.gloves)
+			if(do_after(hooman, HAS_TRAIT(hooman, TRAIT_RIGGER) ? 2 SECONDS : 10 SECONDS, target = src))
+				qdel(src)
+		else
+			return
+
+//stuff to apply mood event for perverts
+/obj/item/clothing/gloves/shibari_hands/equipped(mob/user, slot)
+	.=..()
+	var/mob/living/carbon/human/hooman = user
+	if(HAS_TRAIT(hooman, TRAIT_ROPEBUNNY))
+		hooman.apply_status_effect(/datum/status_effect/ropebunny)
+
+//same stuff as above but for dropping item
+/obj/item/clothing/gloves/shibari_hands/dropped(mob/user, slot)
+	.=..()
+	var/mob/living/carbon/human/hooman = user
+	if(HAS_TRAIT(hooman, TRAIT_ROPEBUNNY))
+		hooman.remove_status_effect(/datum/status_effect/ropebunny)
+
+/obj/item/clothing/shoes/shibari_legs
+	name = "Shibari arms bondage"
+	desc = "Bondage ropes that cover arms."
+	icon_state = "shibari_legs"
+	icon = 'modular_skyrat/modules/modular_items/lewd_items/icons/obj/lewd_clothing/lewd_shoes.dmi'
+	worn_icon = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_clothing/lewd_shoes.dmi'
+	worn_icon_digi = 'modular_skyrat/modules/modular_items/lewd_items/icons/mob/lewd_clothing/lewd_shoes_digi.dmi'
+	body_parts_covered = NONE
+	strip_delay = 100
+	mutant_variants = STYLE_DIGITIGRADE|STYLE_TAUR_ALL
+	slowdown = 4
+	item_flags = DROPDEL
+	///color
+	var/current_color = "pink"
+	///tightness of the ropes can be low, medium and hard.
+	var/tight = "low"
+
+/obj/item/clothing/shoes/shibari_legs/Destroy()
+	var/obj/item/stack/shibari_rope/ropes = new(get_turf(src))
+	ropes.current_color = current_color
+	ropes.update_icon_state()
+	ropes.update_icon()
+	. = ..()
+
+//customization stuff
+/obj/item/clothing/shoes/shibari_legs/update_icon_state()
+	. = ..()
+	icon_state = "[initial(icon_state)]_[current_color]"
+	worn_icon_state = "[initial(icon_state)]_[current_color]"
+
+/obj/item/clothing/shoes/shibari_legs/ComponentInitialize()
+	. = ..()
+	AddElement(/datum/element/update_icon_updates_onmob)
+
+/obj/item/clothing/shoes/shibari_legs/Initialize()
+	. = ..()
+	update_appearance()
+
+//unequip stuff for adding rope to hands
+/obj/item/clothing/shoes/shibari_legs/attack_hand(mob/user)
+	. = ..()
+	if(.)
+		return
+	if(iscarbon(user))
+		var/mob/living/carbon/human/hooman = user
+		if(src == hooman.shoes)
+			if(do_after(hooman, HAS_TRAIT(hooman, TRAIT_RIGGER) ? 2 SECONDS : 10 SECONDS, target = src))
+				qdel(src)
+		else
+			return
+
+//stuff to apply mood event for perverts
+/obj/item/clothing/shoes/shibari_legs/equipped(mob/user, slot)
+	.=..()
+	var/mob/living/carbon/human/hooman = user
+	if(HAS_TRAIT(hooman, TRAIT_ROPEBUNNY))
+		hooman.apply_status_effect(/datum/status_effect/ropebunny)
+
+//same stuff as above but for dropping item
+/obj/item/clothing/shoes/shibari_legs/dropped(mob/user, slot)
+	.=..()
+	var/mob/living/carbon/human/hooman = user
+	if(HAS_TRAIT(hooman, TRAIT_ROPEBUNNY))
+		hooman.remove_status_effect(/datum/status_effect/ropebunny)
```

Comment:
What removes the TRAIT_ROPEBUNNY if the shibari stuff gets deleted rather than just dropped like that? I don't want permanent mood buffs/debuffs because you can find a way to not get it removed in-game.